Build/Build.cs

```diff
@@ -97,19 +102,32 @@ public override ISbomGeneratorBuilder ConfigureSbomGenerationInfoBuilder (Soluti
 
   public override void ConfigureProjects (ProjectsBuilder projects)
   {
-    var normalTestConfiguration = new TestConfiguration(
-        DefaultTestExecutionRuntimeFactory.Instance,
-        TestMatrices.Single(e => e.Name == "NormalTestMatrix"),
+    [CanBeNull]
+    static TestConfiguration CreateTestConfiguration (
+        [CanBeNull] TestMatrix testMatrix,
+        ITestExecutionRuntimeFactory testExecutionRuntimeFactory,
+        ImmutableArray<ITestExecutionWrapper> testExecutionWrappers)
+    {
+      return testMatrix != null
+          ? new TestConfiguration(testExecutionRuntimeFactory, testMatrix, testExecutionWrappers)
+          : null;
+    }
```

Comment:
do we need ReSharper nullabilty here, i.e. we don't have NRT enabled in the build?